packages/toolkit/package.json

```diff
@@ -31,13 +31,36 @@
   ],
   "author": "Aragon Association <legal@aragon.org>",
   "license": "GPL-3.0-or-later",
+  "dependencies": {
+    "@aragon/apm": "^3.1.2-beta.2",
+    "@aragon/apps-shared-minime": "^1.0.2",
+    "@aragon/aragen": "^6.0.0-beta.1",
+    "@aragon/abis": "^1.0.0",
+    "aragon-cli-wrapper": "2.0.0",
```

Comment:
Publish a modified version of `@aragon/wrapper@5.0.0-rc.9` that use `@aragon/abis` instead of `@aragon/os`.